src/Form/EventType.php

```diff
@@ -8,16 +8,24 @@
 use Symfony\Component\OptionsResolver\OptionsResolver;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\Extension\Core\Type\DateType;
+use Symfony\Component\Form\Extension\Core\Type\TimeType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
 
 class EventType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
             ->add('name', TextType::class)
-            ->add('description', TextType::class)
+            ->add('description', TextareaType::class)
             ->add('dateStart', DateType::class, [ 'widget' => 'single_text' ])
+            ->add('dateTime', TimeType::class, [
+                'placeholder' => [
+                    'hour' => 'Heure', 'minute' => 'Minute'
+                ]
+            ])
```

Comment:
A DateTime object store a complete date with hours and minute.

Reply:
Ok the changes are made ...